packages/main/src/ColorPalette.js

```diff
@@ -159,6 +194,12 @@ class ColorPalette extends UI5Element {
 			behavior: ItemNavigationBehavior.Cyclic,
 		});
 
+		this._itemNavigationRecentColors = new ItemNavigation(this, {
+			getItemsCallback: () => this.recentColorsElements,
+			rowSize: 5,
```

Comment:
I can see that you are using this value in many places. It is a perfect candidate to be put in static getter which can be used in all of these places.